README.md

```diff
@@ -1,13 +1,13 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
+This weeks project was to create a API using Express, that containes a bunch of RESTful endpoints that returns diffrent data depending on what you are looking for. 
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## Tech Used
 
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Express
+JavaScript
+Node.js
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://my-express-apii.herokuapp.com/ 
```

Comment:
It would be very nice to have some instructions, how to use your API. Remember to make documentation on what endpoints and queries your user can use. 🤓